backend/src/StdLibExecution/Libs/Json.fs

```diff
@@ -410,16 +411,20 @@ let parse
 
       | CustomType.Record (firstField, additionalFields) ->
         let fieldDefs = firstField :: additionalFields
+        let enumerated = j.EnumerateObject() |> Seq.toList
+
         let dvalMap =
-          j.EnumerateObject()
-          |> Seq.map (fun jp ->
```

Comment:
Previously, this would error if the JSON being parsed had any _extra_ fields, but would not error when fields were missing.
This inverts that a bit - all fields defined in the Record type are required and _extra_ fields in the JSON object are ignored.